vagrant/post-deploy.d/00-run-docker.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eu
+
+PAAS_DIR=/tmp/paas-bootstrap
+
+sudo apt update && sudo apt install docker-compose -y
+
+git clone https://github.com/alphagov/paas-bootstrap.git ${PAAS_DIR}
+cd ${PAAS_DIR}
+git checkout "${BRANCH}"
+cd docker
+
+mkdir -p keys/web keys/worker
+
+ssh-keygen -t rsa -f ./keys/web/tsa_host_key -N ''
+ssh-keygen -t rsa -f ./keys/web/session_signing_key -N ''
+ssh-keygen -t rsa -f ./keys/worker/worker_key -N ''
+
+cp ./keys/worker/worker_key.pub ./keys/web/authorized_worker_keys
+cp ./keys/web/tsa_host_key.pub ./keys/worker
+
+# shellcheck disable=SC2091
+$("${PAAS_DIR}/concourse/scripts/environment.sh")
+
+sudo -E docker-compose up &
```

Comment:
You can use switch `-d` so docker will be in daemon mode after it downloads the images. That way you can avoid the `&` and docker is ready when the script exits.
We still need to wait for ATC to start.